bin/.travis/init_solr.sh

```diff
@@ -45,7 +45,7 @@ fi
 download() {
     case ${SOLR_VERSION} in
         # PS!!: Append versions and don't remove old once, kernel uses this script!
-        7.7.2 )
+        7.7.2|8.5.2 )
```

Comment:
Also maybe we can simplify this to take advantage of bash semantics for matching patterns?

Not tested, and not sure what's the right semantic but something like:
```suggestion
        7.7.*|8.4.*|8.5.* )
```